src/test/java/com/spotify/autoscaler/AutoscalerTest.java

```diff
@@ -114,7 +114,7 @@ public void testTwoClustersFoundAndProcessed() throws IOException {
     when(database.updateLastChecked(cluster1)).thenReturn(true).thenReturn(false);
     when(database.updateLastChecked(cluster2)).thenReturn(true).thenReturn(false);
 
-    final Autoscaler autoscaler =
```

Comment:
why delete all the `final`s ?

Reply:
I had changed the autoscaler reference to avoid code replication since we are always doing the same thing with a different cluster
`new Autoscaler(
            autoscaleJobFactory,
            executorService,
            registry,
            stackDriverClient,
            database,
            sessionProvider,
            clusterStats,
            differentCluster)
`

Then I rolled back this change and forgot to rollback the final as well. I'll fix it, thanks for pointing it out :)